pkg/reconciler/ingress/lister_test.go

```diff
@@ -500,9 +500,104 @@ func TestListProbeTargets(t *testing.T) {
 			},
 		},
 		results: []status.ProbeTarget{{
-			PodIPs: sets.NewString("1.1.1.1"),
-			Port:   "80",
-			URLs:   []*url.URL{{Scheme: "http", Host: "foo.bar.com:80"}},
+			PodIPs:  sets.NewString("1.1.1.1"),
+			PodPort: "80",
```

Comment:
Can we use different port values to ensure the passing is done correcty?

Reply:
Sure, updated.